webapp/src/main/java/com/box/l10n/mojito/service/tm/TMService.java

```diff
@@ -709,7 +710,11 @@ public String generateLocalized(
 
         driver.addStep(new RawDocumentToFilterEventsStep());
         driver.addStep(new CheckForDoNotTranslateStep());
-        driver.addStep(new TranslateStep(asset, repositoryLocale, InheritanceMode.USE_PARENT));
+        if (bcp47Tag.equals("en-x-psaccent")) {
+            driver.addStep(new PseudoLocalizeStep());
+        } else {
+            driver.addStep(new TranslateStep(asset, repositoryLocale, InheritanceMode.USE_PARENT));
```

Comment:
If we use 2 different commands I would refactor slightly `TMService`. I don't think it is super clean to start adding too much logic from 2 different features in that function, specially since Pseudoloc doesn't need a big chuck of it.

I'd introduce `generateLocalizedBase(BasePipelineStep step, String content, String bcp47tag, FilterConfigIdOverride filterConfigIdOverride)` that contains the end of `generatedLocalized` starting at the pipeline configuration. `generateLocalized` will pass down `new TranslateStep(asset, repositoryLocale, InheritanceMode.USE_PARENT)`  as step and have the logic for output tag. For pseudo, have another function `generatePseudolocalized` that pass down the `PseudoLocalizeStep` into `generateLocalizedBase`